shared/api-definitions-ex.asciidoc

```diff
@@ -0,0 +1,70 @@
+[[sample-object]]
+=== Object1 API objects
+++++
+<titleabbrev>Object1</titleabbrev>
+++++
+// ***************************************
+// These pages complement API reference pages. They provide details about the
+// objects in API request bodies or response bodies.
+// These pages are typically nested under an API definitions page.
+// For example, see
+// https://www.elastic.co/guide/en/cloud-enterprise/current/definitions.html
+// ***************************************
+
+//Provide a brief description
+
+Is a cool thing.
+
+// Guidelines for API object documentation
+// ***************************************
+// * Use a definition list.
+// * Each property should be marked as Optional or Required.
```

Comment:
I think we shouldn't put "Optional" or "Required" here, since it will depend on which API is using the object. E.g. it might be required in the context of a PUT xxx API but not for a GET xxx API

Reply:
Good point. +1 for removing.